<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1040</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.112, y1:0.107318, x2:1, y2:0.994, stop:0 rgba(202, 216, 225, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Noto Mono&quot;;
color: rgb(52, 52, 52);</string>
     </property>
     <property name="text">
      <string>4IND</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="userLabel">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(46, 52, 54);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chatWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1038</x>
      <y>0</y>
      <width>420</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 200);</string>
    </property>
    <widget class="QPlainTextEdit" name="chatEdit">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>340</width>
       <height>850</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(46, 52, 54);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputEdit">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>900</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>900</y>
       <width>52</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(46, 52, 54);</string>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
